README.md

```diff
@@ -3,9 +3,9 @@
 ## Overview
 
 nrfx is an extract from the nRF5 SDK that contains solely the drivers for
```

Comment:
nrfx was extracted, but now I think we should change this overview to focus that there are only standalone drivers without dependencies

Reply:
Again, I took the text [from the Wiki](https://github.com/NordicSemiconductor/nrfx/wiki/Overview) without any modifications.
I do not think we need a Wiki because all information could be in the Readme and Contributing guide. What do you think?